sw/device/silicon_creator/manuf/extensions/example_personalize_ext.c

```diff
@@ -8,6 +8,7 @@
 #include "sw/device/lib/testing/test_framework/status.h"
 #include "sw/device/lib/testing/test_framework/ujson_ottf.h"
 #include "sw/device/silicon_creator/lib/cert/cert.h"
+#include "sw/device/silicon_creator/manuf/base/personalize_ext.h"
```

Comment:
I think you need to modify the inputs to the two perso extension functions in this file too.

Reply:
done.